crypto/evp_extra/pq_sig_kat_tests_dilithium3.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <openssl/evp.h>
+#include "../rand_extra/pq_custom_randombytes.h"
```

Comment:
nit: move this include after vector

Reply:
Thanks, fixed in https://github.com/awslabs/aws-lc/pull/637/commits/ad38c62541c964623f04455185f6b40639e80eeb.